nextflow.config

```diff
@@ -54,6 +56,17 @@ params {
   bismark_align_mem_per_multicore = null
   publish_dir_mode = 'copy'
 
+  bwa_biscuit_index =  false 
+  whitelist = false
+  blacklist = false 
+  common_dbsnp = false
+  save_pileup_file = false
```

Comment:
Maybe we could  reuse `params.save_align_intmeds` here for both `save_pileup_file` and `save_snp_file` ?

Reply:
From my experience, the SNP file is needed in most cases, so I think maybe we should leave the `save_snp_file` but we can probably merge the `save_pileup_file` with `save_align_intermidates`.
What do you think?